ArcGISRuntimeSDKQt_QMLSamples/Layers/ClassBreaksWithAlternateSymbols/ClassBreaksWithAlternateSymbols.qml

```diff
@@ -0,0 +1,172 @@
+            top: parent.top
+            margins: 15
+        }
+        spacing: 5
+
+        Button {
+            width: 200
+            enabled: true
+            text: "Set Scale to 2500"
+            onClicked:
+            {
+                const scale = 2500;
+                mapView.setViewpointCenterAndScale(ArcGISRuntimeEnvironment.createObject("Point", {x: -13632095.660131, y: 4545009.846004}), scale);
+            }
+        }
+
+        Button {
+            width: 200
+            enabled: true
+            text: "Set Scale to 7500"
+            onClicked:
+            {
+                const scale = 7500;
+                mapView.setViewpointCenterAndScale(ArcGISRuntimeEnvironment.createObject("Point", {x: -13632095.660131, y: 4545009.846004}), scale);
+            }
+        }
+
+        Button {
+            width: 200
+            enabled: true
+            text: "Set Scale to 15000"
+            onClicked:
+            {
+                const scale = 15000;
+                mapView.setViewpointCenterAndScale(ArcGISRuntimeEnvironment.createObject("Point", {x: -13632095.660131, y: 4545009.846004}), scale);
+            }
+        }
+    }
+
+    // Javascript code goes here
+    function createClassBreaksRenderer_() {
+
+
+        // create the default symbol
+        const purple_diamond = ArcGISRuntimeEnvironment.createObject("SimpleMarkerSymbol", {
+                                                                             color: "purple",
+                                                                             size: 30,
+                                                                             style: Enums.SimpleMarkerSymbolStyleDiamond
+                                                                         });
+        const purple_diamond_ml = purple_diamond.toMultilayerSymbol();
+
+        const red_triangle = ArcGISRuntimeEnvironment.createObject("SimpleMarkerSymbol", {
+                                                                             color: "red",
+                                                                             size: 30,
+                                                                             style: Enums.SimpleMarkerSymbolStyleTriangle
+                                                                         });
+        const red_triangle_ml = red_triangle.toMultilayerSymbol();
+        red_triangle_ml.referenceProperties = ArcGISRuntimeEnvironment.createObject("SymbolReferenceProperties", {minScale: 5000, maxScale: 0});
+
+        const alternate_symbols = createAlternateSymbols_();
+
+        let class_break = ArcGISRuntimeEnvironment.createObject("ClassBreak", {label: "CB1", description: "CB1", minValue: 0, maxValue: 1, symbol: red_triangle_ml});
+
+        class_break.alternateSymbols.append(alternate_symbols[0]);
+        class_break.alternateSymbols.append(alternate_symbols[1]);
+
+        //create the class breaks renderer using the class break created above
```

Comment:
```suggestion
        // Create the class breaks renderer using the class break created above
```
"Begin comment text with an uppercase letter (with a single space between the comment characters and the comment text)."